src/Raven.Client/Util/ConnectionStringParser.cs

```diff
@@ -89,17 +74,6 @@ protected virtual bool ProcessConnectionStringOption(ConnectionStringOptions opt
                 case "apikey":
                     options.ApiKey = value;
                     break;
-                case "user":
-                    networkCredentials.UserName = value;
-                    _setupUsernameInConnectionString = true;
-                    break;
-                case "password":
-                    networkCredentials.Password = value;
-                    _setupPasswordInConnectionString = true;
-                    break;
-                case "domain":
-                    networkCredentials.Domain = value;
-                    break;
                 case "urls":
```

Comment:
@ayende Yes, I will add this request to http://issues.hibernatingrhinos.com/issue/RavenDB-7286